### DISCUSSION TITLE
推荐视频的右键功能菜单 打开位置可能不在屏幕内

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#606] [bug]
### Bug 描述

在靠近屏幕左侧边界的位置点击右键，控制菜单会部分展示在屏幕外
在靠近屏幕右侧边界的位置点击右键，控制菜单会向屏幕内偏移较多位置

### 操作步骤

```plain text
1.打开应用(版本为v4.6.30.0)
2.在首页-推荐 右键单击视频
3.在特定区域右键单击，展开的菜单可能无法完整展示
```

### 期望结果

右键菜单不会超出屏幕

### 截图

![repo1](https://github.com/ywmoyue/biliuwp-lite/assets/135402717/a3345770-6559-4537-a06a-10c0f917c320)
![repo2](https://github.com/ywmoyue/biliuwp-lite/assets/135402717/3257c5dd-aecd-457c-9222-a122399b2b98)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 24H2 (Build 26200.5001)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
确实有这个问题，感谢反馈

**Comment by ywmoyue** [comment]
怎么close了，这个问题还没处理呀

**Reply by TPAXcc** [reply]
😂
